components/esp32/Kconfig

```diff
@@ -117,6 +117,11 @@ config SYSTEM_EVENT_TASK_STACK_SIZE
     help
         Config system event task stack size in different application.
 
+config TCPIP_TASK_STACK_SIZE
+    int "Tcp IP loop task stack size"
```

Comment:
Suggest calling this "TCP/IP Task Stack Size"

Under "help" suggest this text:

```
Configure TCP/IP task stack size, used by LWIP to process multi-threaded TCP/IP operations. The default is 2048 bytes, setting this stack too small will result in stack overflow crashes.
```